runtime/j9vm/java11vmi.c

```diff
@@ -1470,16 +1473,22 @@ JVM_GetVmArguments(JNIEnv *env) {
 
 		vmJniClass = (jclass)internalFunctions->j9jni_createLocalRef(env, vmClass->classObject);
 
+		/* exit vm before calling jni */
+		internalFunctions->internalExitVMToJNI(currentThread);
+
 		mid = (*env)->GetStaticMethodID(env, vmJniClass, "getVMArgs", "()[Ljava/lang/String;");
```

Comment:
To avoid getting & releasing vmaccess around every call, you can use the VM version of getMethodID:
```
(jmethodID) vmFunctions->getJNIMethodID(vmThread, method);
```

This doesn't do the static check for you but given it's VM code, seems safe enough to me.